src/pages/chatPage.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function chatPage() {
+  return <div>chat</div>;
+}
```

Comment:
컴포넌트 PascalCase 로 작성하는 것은 스스로 정한 원칙 아니였나요?

```suggestion
function ChatPage() {
  return <div>chat</div>;
}
```

Reply:
Page == Component